admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -90,6 +90,7 @@ module.exports = {
     attachment: './js/pages/attachment',
     attribute: './js/pages/attribute',
     attribute_group: './js/pages/attribute-group',
+    address: './js/pages/address',
```

Comment:
Could you sort this hash please?
At least you'll never have a conflict again (or less).